api-operator/pkg/controller/integration/integration_config.go

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2020 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
```

Comment:
Shall we change the license year as 2021?

Reply:
Done.